src/exception/PositionNotFoundException.kt

```diff
@@ -0,0 +1,7 @@
+package exception
+
+class PositionNotFoundException(override val message: String = NO_SUCH_COORDINATE_EXCEPTION_MESSAGE) : InvalidMoveException(message) {
```

Comment:
minor: unlike java, in kotlin all these little classes could be in a single file

Reply:
ok